pyqtgraph/opengl/items/GLTextItem.py

```diff
@@ -78,6 +78,8 @@ def paint(self):
 
         text_pos = self.__project(self.pos, modelview, projection, viewport)
         text_pos[1] = viewport[3] - text_pos[1]
+
+        text_pos /= self.view().devicePixelRatio()
```

Comment:
I solved the error by dividing the coordinates by the pixel ratio of the device !